upup/pkg/fi/cloudup/vspheretasks/attachiso.go

```diff
@@ -57,6 +94,73 @@ func (_ *AttachISO) CheckChanges(a, e, changes *AttachISO) error {
 }
 
 func (_ *AttachISO) RenderVC(t *vsphere.VSphereAPITarget, a, e, changes *AttachISO) error {
-	glog.Info("AttachISO.RenderVC invoked!")
+	startupScript, err := changes.BootstrapScript.ResourceNodeUp(changes.IG)
+	startupStr, err := startupScript.AsString()
+	if err != nil {
+		return fmt.Errorf("error rendering startup script: %v", err)
+	}
+	dir, err := ioutil.TempDir("", *changes.VM)
```

Comment:
This gets created in the temp directory of the OS. For unix based systems, this is in /tmp directory. Yes, all the files including the folder we create gets cleaned up once AttachISO.Render* method completes.